replay_client.py

```diff
@@ -0,0 +1,841 @@
+'''
```

Comment:
This file too, is in the middle of the repository. This has to be formatted as a primitive and placed in one directory with the rest of them (`centinel/primitives`).
Additionally, this hasn't been cleaned at all. I still see a lot of debug flags and other unused parameters that haven't been cleaned up.